### PR TITLE
CJ4: Fixes Battery Amps Display in ECIS and SYS pages.  From what I could …

### PR DESCRIPTION
**Pull request by Danice737**
…tell seems battery load is actaully tracked in WATTs not amps, appling the Watts to Amps conversion formula seems to display correct values.

### CODE REVIEW

File: src/workingtitle-aircraft-cj4/SimObjects/Airplanes/Asobo_CJ4/systems.cfg [hunk 1]
```diff
@@ -178,8 +178,8 @@ max_pitch_velocity_hi_alt =0.2
 max_pitch_velocity_lo_alt_breakpoint =20000
 max_pitch_velocity_hi_alt_breakpoint =28000
 max_bank = 30,15
-max_bank_acceleration = 1.5
-max_bank_velocity = 5
+max_bank_acceleration = 1.7
```
Comment (mattnischan) [inline src/workingtitle-aircraft-cj4/SimObjects/Airplanes/Asobo_CJ4/systems.cfg:181]: I think these changes snuck in by accident.

**Review by mattnischan — CHANGES_REQUESTED** [review]
@Danice737 Thanks for another contribution! I'll have to check the docs on this watts/amps units issue, but that does seem plausible.

File: .vs/VSWorkspaceState.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (mattnischan) [inline .vs/VSWorkspaceState.json:1]: I think these VS files are also here by mistake.

Comment (Danice737) [reply .vs/VSWorkspaceState.json:1]: Not sure why it did that

File: src/workingtitle-aircraft-cj4/SimObjects/Airplanes/Asobo_CJ4/systems.cfg (hunk 1 above)
Comment (Danice737) [reply src/workingtitle-aircraft-cj4/SimObjects/Airplanes/Asobo_CJ4/systems.cfg:181]: if you look they comment all the power as watts in the systems.cfg but list the variable as amps in SDK
image

Comment (Danice737) [reply src/workingtitle-aircraft-cj4/SimObjects/Airplanes/Asobo_CJ4/systems.cfg:181]: this was a second PULL those fixe the AP hunting the NAV or ILS (flying back and forth across your trk)

Comment (Danice737) [reply src/workingtitle-aircraft-cj4/SimObjects/Airplanes/Asobo_CJ4/systems.cfg:181]: > I think these changes snuck in by accident.

this was a second PULL those fix the AP hunting the NAV or ILS (flying back and forth across your trk)

Comment (Danice737) [reply src/workingtitle-aircraft-cj4/SimObjects/Airplanes/Asobo_CJ4/systems.cfg:181]: not sure why Git added it here when I pushed it separately this is for a AP change

Comment (mattnischan) [reply src/workingtitle-aircraft-cj4/SimObjects/Airplanes/Asobo_CJ4/systems.cfg:181]: I'm not sure how familiar you are with Git so I apologise if I'm staying things you already know. Pull Requests are by branch, so if you pushes both commits to the same branch then they would appear here.

To make a separate pull request you would need to make another branch, push those changes to that branch, and submit that branch for pull request.

**Comment by mattnischan** [comment]
There are a few more VS specific metadata files that are still included.

File: src/workingtitle-aircraft-cj4/SimObjects/Airplanes/Asobo_CJ4/systems.cfg (hunk 1 above)
Comment (Danice737) [reply src/workingtitle-aircraft-cj4/SimObjects/Airplanes/Asobo_CJ4/systems.cfg:181]: Ah I had forgot that, thank you it has been 6 yrs a little rusty :) Should I create that in a new pull and resubmit it for change or can it get commited here this time?

Comment (Danice737) [reply src/workingtitle-aircraft-cj4/SimObjects/Airplanes/Asobo_CJ4/systems.cfg:181]: Btw I excluded the VS directory so they shouldn't show up next time :)

Comment (mattnischan) [reply src/workingtitle-aircraft-cj4/SimObjects/Airplanes/Asobo_CJ4/systems.cfg:181]: No worries! Yeah, if you wouldn't mind submitting separately, that would be awesome. Normally I wouldn't be super strict about stuff like that in this case, but for flight model adjustments it's good to be able to test that separately as those changes can be finicky.

Comment (Danice737) [reply src/workingtitle-aircraft-cj4/SimObjects/Airplanes/Asobo_CJ4/systems.cfg:181]: Ok will do.

**Comment by Danice737** [comment]
> There are a few more VS specific metadata files that are still included.

I can't get it to remove the bin files..  I can delete the whole pull and start over.

**Comment by mattnischan** [comment]
So, if you're doing it from inside VS, it may need those files and so you may be fighting the IDE. Try removing them from the command line before committing.

Also, sorry to keep requesting changes, but it appears you deleted the systems.cfg file instead of just reverting the changes, so now the PR change wants to delete the file from the repo as well.

**Comment by Danice737** [comment]
Redoing the whole pull

**Pull request closed without merging**